### DISCUSSION TITLE
relative URL issue after clicking "next page" in the main page

### DISCUSSION THREAD

**Discussion by yseeker** [adityatelange/hugo-PaperMod#770] [Q&A]
The main page of my web is at something like https://something.com/blog, but somehow it goes to https://something.com/page/2 after clicking the "next page" on the footer, which is supposed to go to https://something.com/**blog**/page/2. Does anyone know how can I fix this issue? The exampleSite looks have no such an issue.

I am using the configuration as below, except these other all config are the same as the default as shown in exampleSite:

baseURL: https://something.com/blog/
title: something
publishDir: "docs"
canonifyurls: true
theme: PaperMod

**Comment by adityatelange** [comment]
set proper `baseURL: https://something.com/blog/`

**Reply by yseeker** [reply]
I already did that. But let me double check.

**Reply by yseeker** [reply]
Solved. Interestingly, by changing the baseURL from the correct URL with custom domain (https://something.com/blog/) to the original github URL (https://something.github.io/blog/), the issue was fixed. Thanks anyway.
